Tests/FluentMongoTests/DriverTests.swift

```diff
@@ -0,0 +1,88 @@
+import XCTest
+
+import Fluent
+@testable import FluentMongo
+
+/**
+    To run these tests you must have `mongod`
+    running with the following configuration.
+ 
+    - user: test
+    - password: test
+    - localhost: test
+    - port: 27017
+*/
+class DriverTests: XCTestCase {
+    static var allTests : [(String, (DriverTests) -> () throws -> Void)] {
+        return [
+            ("testConnectFailing", testConnectFailing),
+            ("testSaveAndClearUsers", testSaveAndClearUsers),
+            ("testSaveAndFind", testSaveAndFind)
+        ]
+    }
+    
+    
+    var database: Fluent.Database!
+    var driver: MongoDriver!
+    
+    override func setUp() {
+        driver = MongoDriver.makeTestConnection()
```

Comment:
Using the same driver may become an issue if we get more tests as they run concurrently. It's pretty easy to run into race conditions, etc. Might be worth it to create a connection for each test.